RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
```

Comment:
I don't think there is need for a release. This workflow lives only on github.

Reply:
I came here to say the same thing and agree that the release file should be removed from this PR.

Reply:
Understood, file removed.